pubnub-kotlin/pubnub-kotlin-api/src/appleMain/kotlin/com/pubnub/api/PubNubImpl.kt

```diff
@@ -353,14 +353,18 @@ class PubNubImpl(private val pubNubObjC: KMPPubNub) : PubNub {
         channels: List<String>,
         channelGroups: List<String>,
         includeState: Boolean,
-        includeUUIDs: Boolean
+        includeUUIDs: Boolean,
+        limit: Int,
```

Comment:
Isn't limit optional, and if missing, assigned to default to enforce maximum value?
If this function is not called directly by the user, then it shouldn't be a problem.